src/stac_iitmandi/templates/stac_iitmandi/_navbar.html

```diff
@@ -1,27 +1,40 @@
-
+{% load static %}
 <nav class="navbar navbar-expand-lg sticky-top bg-custom navbar-light mt-5">
-    <a class="navbar-brand" href="#">STAC</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+<!--<nav class="navbar navbar-expand-lg navbar-dark bg-primary">-->
```

Comment:
We don't need this comment.

Reply:
fixed in 96587951fe63db5be7385b1b9f60e6ba12312d21